src/core/connection.c

```diff
@@ -1635,8 +1635,9 @@ QuicConnTryClose(
         } else {
             QuicTraceEvent(
                 ConnTransportShutdown,
-                "[conn][%p] Transport Shutdown: %llu (Remote=%hhu) (QS=%hhu)",
+                "[conn][%p] Transport Shutdown: %u (%llu) (Remote=%hhu) (QS=%hhu)",
```

Comment:
ETW compiler disallows mapping on values larger than uint32, so in order to get textual mapping, we need to log the error code twice, once as a 32-bit unsigned integer to allow for mapping, and again as a 64-bit unsigned integer in case the error uses the high DWORD.